modules/op-pipelines-as-code-command-reference.adoc

```diff
@@ -0,0 +1,109 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="pipelines-as-code-command-reference_{context}"]
+= {pac} command reference 
+
+[role="_abstract"]
+The `tkn-pac` CLI tool offers the following capabilities:
+
+* Bootstrap {pac} installation and configuration.
+* Create a new {pac} repository.
+* List all {pac} repositories.
+* Describe a {pac} repository and the associated runs.
+* Generate a simple pipeline run to get started.
+* Resolve a pipeline run as if it was executed by {pac}.
```

Comment:
Needs more explanation to convey that this is in order to run the pipeline without having to make a change in the Git repository, for example for testing purposes